src/main/java/core/basesyntax/model/Activities.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.model;
+
+public enum Activities {
```

Comment:
naming convention: https://mate-academy.github.io/style-guides/java/java.html#s4.8.1-enum-classes
```suggestion
public enum Activity {
```
OR
```suggestion
public enum Operation {
```